### ISSUE TITLE
vault api error

### ISSUE THREAD

**Issue by nstringham** [SkyCryptWebsite/SkyCrypt#156]
**Describe the bug**
SkyCrypt incorrectly says that I have disabled the API for vault

**Profile/Location**
https://sky.shiiyu.moe/stats/Cookie_Wookie_7/Tomato

**Additional Info**
I have never used my vault

**Expected behavior**
The notice should not appear when the API is enabled

**Screenshots**
![image](https://user-images.githubusercontent.com/44071655/99866670-00417f00-2b81-11eb-8b89-28ac5a3b4b10.png)
![image](https://user-images.githubusercontent.com/44071655/99866673-020b4280-2b81-11eb-918c-f662dc2a7fbb.png)

**Comment by metalcupcake5** [comment]
Could be because you haven't unlocked the personal vault, or because there is nothing in the personal vault.

**Comment by nstringham** [comment]
> Could be because you haven't unlocked the personal vault, or because there is nothing in the personal vault.

Yes probably but the error message should be changed in that case

**Comment by metalcupcake5** [comment]
ok pretty sure the api is just wack

**Comment by nstringham** [comment]
I think the problem is that SkyCrypt assumes that just because there is no vault data the api is disabled

**Comment by WarpWing** [comment]
@nstringham This still happening?

**Comment by nstringham** [comment]
> @nstringham This still happening?

Hard to say because I no longer have a profile that hasn't unlocked the vault but I don't think this has been fixed so it's probably still broken

**Comment by WarpWing** [comment]
> > @nstringham This still happening?
> 
> Hard to say because I no longer have a profile that hasn't unlocked the vault but I don't think this has been fixed so it's probably still broken

It could have been fixed in a later update. If you have a friend or a profile that is close, would be dope if you could check. This should have been patched.
